### DISCUSSION TITLE
作者可以考虑下雷电模拟器4.0.46版本模拟器

### DISCUSSION THREAD

**Discussion by Ren3030** [dundunnp/hamibot-auto_xuexiqiangguo#646] [Ideas]
经过我试用发现这个版本的4.0.46模拟器使用修改版2.33强国可以使用
四人双人答题无异常，模拟器分辨率要设置对，不然识别不到文字题目
缺点就是使用2.33修改版会出现异常情况不出现滑条及网络异常，遇到这种情况就只能重启强国APP
没遇到这种情况很顺畅

**Comment by dundunnp** [comment]
有时间我试下

**Reply by Ren3030** [reply]
2月15日凌晨发现新验证滑块2.33改版没截图下来，可能以后要改滑块验证这方面了
